test/org/sagebionetworks/bridge/TestUtils.java

```diff
@@ -17,7 +23,22 @@ public void run() {
             }
         }
     }
-    
+
+    public static Http.Context mockPlayContextWithJson(String json) throws Exception {
```

Comment:
Credit goes to Alx. He wrote the original method. I just ported it to the shared TestUtils class.